### PR TITLE
chore(build): Upgrade to Typescript v3.9.2

### PR DESCRIPTION
**Pull request by GordonSmith**
Add downgraded types for customers on older versions.

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review - you will find it easier with the w=1 option:  https://github.com/hpcc-systems/Visualization/pull/3603/files?w=1

File: demos/gallery/src-umd/playground.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ var __extends = (this && this.__extends) || (function () {
 })(function (require, exports) {
     "use strict";
     Object.defineProperty(exports, "__esModule", { value: true });
+    exports.App = void 0;
```
Comment (jbrundage) [inline demos/gallery/src-umd/playground.js:25]: What's this?

File: packages/chart/package.json [hunk 2]
```diff
@@ -1,84 +1,94 @@
...
+        "build": "npm run compile-es6 && npm run bundle",
+        "watch": "npm-run-all compile-es6 -p compile-es6-watch bundle-watch",
+        "stamp": "node node_modules/@hpcc-js/bundle/src/stamp.js",
+        "lint": "tslint --project . src/**/*.ts",
+        "docs": "typedoc --options tdoptions.json ."
+    },
+    "dependencies": {
+        "@hpcc-js/api": "^2.8.20",
+        "@hpcc-js/common": "^2.26.0",
+        "@hpcc-js/util": "^2.14.0"
```
Comment (jbrundage) [inline packages/chart/package.json:42]: Lowered the version here? Was that intentional?

File: demos/gallery/src-umd/playground.js (hunk 1 above)
Comment (GordonSmith) [reply demos/gallery/src-umd/playground.js:25]: Its generated code.

File: packages/chart/package.json (hunk 2 above)
Comment (GordonSmith) [reply packages/chart/package.json:42]: This will get auto "corrected" during the next release cycle.

**Pull request merged**